### DISCUSSION TITLE
Transcoding for Samsung UE32ES5500

### DISCUSSION THREAD

**Discussion by FFDA** [gerbera/gerbera#2360] [Q&A]
Hello,

**TL;DR: I’m looking for someone to share a working transcoding profile that works for this TV or similar age Samsung TV.**

I’m trying to set up transcoding for one of my Samsung TVs. It’s an old unit from 2012 - Samsung UE32ES5500.
Gerbera working perfectly for this TV right now, except that it does not support newer video/audio formats that where released in last decade. I can play x264 files, but it doesn’t support x265 format that my library is mostly made of and I plan to use it in the future.
I’m looking for someone to share a working transcoding profile that works for this TV or similar age Samsung TV.
I think the problem is that I do not set any dlna profiles in transcoding options, but I’m out of my depth here.
Just to illustrate my issue I have a flv file that can be played streaming from gerbera without any issues.  If I change containers for the same file with command: `ffmpeg -i filename.flv -acodec copy -vcodec copy -f matroska filaname.mkv` and put it in USB drive - it plays without issues, but if I to play this file with transcoding turned on in gerbera with this profile
```
       </profile>
          <profile name="test2" enabled="yes" type="external">
          <mimetype>video/x-matroska</mimetype>
          <accept-url>yes</accept-url>
          <first-resource>yes</first-resource>
          <hide-original-resource>yes</hide-original-resource>
          <agent command="ffmpeg" arguments="-i %in -acodec copy -vcodec copy -f matroska %out"/>
          <buffer size="10485760" chunk-size="262144" fill-size="524288"/>
        </profile>
```
 it fails with error message that file format is not supported.
